src/main/java/com/kobylynskyi/graphql/codegen/generators/FreeMarkerTemplateFilesCreator.java

```diff
@@ -48,7 +48,13 @@ public static File create(MappingContext mappingContext,
         }
 
         try (FileWriter fileWriter = new FileWriter(javaSourceFile)) {
-            Template template = FreeMarkerTemplatesRegistry.getTemplateWithLang(language, templateType);
+            Template template;
+            if (mappingContext.getCustomTemplates().containsKey(templateType)) {
+              template = FreeMarkerTemplatesRegistry.getCustomTemplates(mappingContext.getCustomTemplates().get(templateType));
```

Comment:
1 - Don't make 2 map lookups (first `.containsKey()`, then `.get()`). Instead - do `.get()` and then check if the value is `null`.
2 - Use constant indentation of 4 spaces.